src/ivcam/sr300.cpp

```diff
@@ -57,7 +70,8 @@ namespace librealsense
         std::vector<std::shared_ptr<device_info>> results;
 
         auto correct_pid = filter_by_product(uvc, { SR300_PID, SR300v2_PID });
-        auto group_devices = group_devices_by_unique_id(correct_pid);
+     
+		auto group_devices = group_devices_by_unique_id(correct_pid);
```

Comment:
Remove extra spaces/tabs

Reply:
This also highlighted in Travis log:
>**[ERROR] src/ivcam/sr300.cpp has tabs (this project is using spaces as delimiters)**
**[ERROR] src/ivcam/sr300.h has tabs (this project is using spaces as delimiters)**
find: ‘third-party/libtm’: No such file or directory
**Pull-Request check failed, please address 2 the errors reported above**

https://travis-ci.org/IntelRealSense/librealsense/jobs/660036200?utm_medium=notification&utm_source=github_status